setup.py

```diff
@@ -17,6 +20,26 @@ def run_tests(self):
         errno = pytest.main(self.test_args)
         sys.exit(errno)
 
+
+class install_lib(_install_lib):
+    """
+        install_lib class extended to compile the locales
+        from the .po source files in .mo files.
+    """
+    def run(self):
+        _install_lib.run(self)
+        try:
+            # Use the management command to compile messages
+            # tested with django 1.7-1.9, 1.9 does not need the chdir anymore
+            from django.core.management.commands.compilemessages import Command
+            os.chdir('registration')
+            cmd = Command()
+            cmd.handle(verbosity=0, exclude=[], locale=[])
```

Comment:
can you use the `call_command` api? https://docs.djangoproject.com/en/1.9/ref/django-admin/#running-management-commands-from-your-code